src/test/clojure/clara/test_rules.clj

```diff
@@ -3876,16 +3926,18 @@
               {:?temps [[] 20]} 1})
           (str "Two Cold facts with single firing for node type " node-type))
 
-      ;; FIXME: The correct assertion here is on equality to {:?temps [[] nil]}.
-      ;; Since (->Cold 10) is retracted we should use the initial value here.
-      ;; This is an existing defect that has been logged at https://github.com/rbrush/clara-rules/issues/188
       (is (= (-> empty-session
                  (insert (->Cold 10) (->Hot 100))
                  (fire-rules)
                  (retract (->Cold 10))
                  (fire-rules)
                  (query q1))
-             [{:?temps [[] 10]}])
+             (if (= "AccumulateWithJoinFilterNode" node-type)
+               [{:?temps [[] 10]}]
+               ;; FIXME: The correct assertion here is on equality to {:?temps [[] nil]}.
+               ;; Since (->Cold 10) is retracted we should use the initial value here.
+               ;; This is an existing defect that has been logged at https://github.com/rbrush/clara-rules/issues/188
```

Comment:
The comment on this PR suggest that 188 should be fixed, but this suggests there is more to do with the AccumulateWithJoinFilterNode. I'm okay with merging this either way, following up on #188 later, since it isn't a regression of current behavior.